### ISSUE TITLE
css_file always uses '/' but '\' is also possible

### ISSUE THREAD

**Issue by Sogolumbo** [pelican-plugins/seo#61]
The path to the css file may have to be interpreted by a browser on a windows machine. The hardcoded `/` characters [here](https://github.com/Sogolumbo/pelican-seo/blob/b1aadb6ab21e6fbc2bbaa08a5a5cdfff35c2d726/pelican/plugins/seo/seo_report/__init__.py#L263) cause an error:
The css file can not be found.

Replacing  [the line](https://github.com/Sogolumbo/pelican-seo/blob/b1aadb6ab21e6fbc2bbaa08a5a5cdfff35c2d726/pelican/plugins/seo/seo_report/__init__.py#L263) with the following code works on Windows:
```
css_file = "file:///" + os.path.join(plugin_path, "static", "seo_report.css")
```

**Comment by MaevaBrunelles** [comment]
Indeed. I suppose that the fix you propose couldn't work on Unix with `"file:///"`. I need some time to do some tests.
